src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -157,14 +284,27 @@ const UserVocabularyProcessScreen = ({ navigation }: UserVocabularyProcessScreen
         />
         <StyledHintText>{maxPictureUpload}</StyledHintText>
         {hasImageErrorMessage && <ContentError>{getLabels().userVocabulary.creation.imageErrorMessage}</ContentError>}
-
-        <AddAudioButton
-          onPress={() => null}
-          label={addAudio}
-          buttonTheme={BUTTONS_THEME.text}
-          iconLeft={MicrophoneCircleIcon}
-          iconSize={theme.spacingsPlain.xl}
-        />
+        {recordingPath ? (
+          <AudioContainer style={{ opacity: fadeAnim }}>
+            <>
+              <AudioText>{recordingPath.substring(recordingPath.lastIndexOf('/') + 1).toUpperCase()}</AudioText>
+              <DeleteContainer onPress={() => deleteRecording()} testID='delete-audio-recording'>
+                <CloseCircleIconBlue width={theme.spacingsPlain.lg} height={theme.spacingsPlain.lg} />
+              </DeleteContainer>
+              <PlayContainer onPress={() => onStartPlay()}>
```

Comment:
Could you keep the button pressed, while the audio is played? Also there is no indication that the button is pressed. Wouldn't the color change to dark if you press the icon?

Reply:
well the outline svg icons don't have a styleable background. I switched to the normal AudioPlayer Icon and used the same color indicators

Reply:
is that fine for the moment? If not i have to get new svg files from the design team

Reply:
we can use the audio player standard icon